tests/test.py

```diff
@@ -364,11 +364,12 @@ def test_prometheus_metrics_worker_restart(self, web_container):
         restart can be observed after the worker restart.
         """
         web_client = web_container.http_client()
-        response = web_client.get('/metrics', headers={'Connection': 'close'})
+        response = web_client.get('/metrics')
 
         [sample] = http_requests_total_for_view(
             response.text, 'prometheus-django-metrics')
         assert_that(sample.value, Equals(1.0))
+        response.close()
```

Comment:
I don't think we need this, given that we already read the whole request body (with `response.text` above) and all it does is release the connection back to the pool for reuse.